app/src/main/java/org/simple/clinic/medicalhistory/ui/MedicalHistoryQuestionItem.kt

```diff
@@ -0,0 +1,159 @@
+package org.simple.clinic.medicalhistory.ui
+
+import androidx.compose.animation.animateColorAsState
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.ChipDefaults
+import androidx.compose.material.Divider
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.FilterChip
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import org.simple.clinic.R
+import org.simple.clinic.common.ui.theme.SimpleTheme
+import org.simple.clinic.medicalhistory.Answer
+import org.simple.clinic.medicalhistory.MedicalHistoryQuestion
+
+@Composable
+fun MedicalHistoryQuestionItem(
+  question: MedicalHistoryQuestion,
+  selectedAnswer: Answer?,
+  modifier: Modifier = Modifier,
+  showDivider: Boolean = true,
+  onSelectionChange: (Answer) -> Unit,
+) {
+  Column(
+    modifier = modifier
+      .background(MaterialTheme.colors.surface)
+  ) {
+    Row(
+      modifier = Modifier,
+      verticalAlignment = Alignment.CenterVertically,
+      horizontalArrangement = Arrangement.spacedBy(dimensionResource(R.dimen.spacing_16))
+    ) {
+      Text(
+        modifier = Modifier.weight(1f),
+        text = stringResource(question.questionRes),
+        style = MaterialTheme.typography.body1,
+        color = MaterialTheme.colors.onSurface,
+        maxLines = 1,
+        overflow = TextOverflow.Ellipsis,
+      )
+
+      AnswerChipsGroup(
+        selectedAnswer = selectedAnswer,
+        onSelectionChange = { newAnswer ->
+          if (newAnswer == selectedAnswer) {
+            onSelectionChange(Answer.Unanswered)
+          } else {
+            onSelectionChange(newAnswer)
+          }
+        }
+      )
+    }
+
+    if (showDivider) {
+      Divider(color = SimpleTheme.colors.onSurface11)
```

Comment:
@msasikanth The default color for divider is `MaterialTheme.colors.onSurface.copy(alpha = DividerAlpha)` where alpha = 0.12f. Maybe we can use a default value only?

Reply:
Our default divider color is surface11, let's just merge it in, we can make those changes through out app later instead of starting from now.